tests/suites/0_stateless/01_system/01_0004_system_engines.sql

```diff
@@ -1 +1 @@
-SELECT * FROM system.engines ORDER BY Engine LIMIT 1,2;
+SELECT * FROM system.engines ORDER BY Engine LIMIT 2 offset 1;
```

Comment:
As far as I know, we need this syntax in some MySQL compatibility. CC: @TCeason

Reply:
Yes, if want to use dbeaver to query a table, the GUI will send query like this:

select * from db.t limit 0,200;

Reply:
Actually we support this syntax, but there is bug that mistake choosing the limit value and offset value. I've just fixed it.